### ISSUE TITLE
Warning: array_flip(): Can only flip STRING and INTEGER values!

### ISSUE THREAD

**Issue by maxpowel** [simplethings/EntityAudit#143]
Hello!

I just installed and configured the bundle but when clearing cache I got this warning:

Warning: array_flip(): Can only flip STRING and INTEGER values! in /Users/maxpowel/erp/vendor/simplethings/entity-audit-bundle/src/SimpleThings/EntityAudit/Metadata/MetadataFactory.php line 32 

My symfony version is:
Symfony version 2.5.12 - app/dev/debug

**Comment by tolry** [comment]
ping @DavidBadura @andrewtch

**Comment by DavidBadura** [comment]
Sorry for the late response. Can you send us your configuration? We can not reproduce this problem.

**Comment by DavidBadura** [comment]
We can not reproduce it. If the problem still exists (in the latest version), please create a new ticket with your configuration.

**Comment by SanderVerkuil** [comment]
It appears that the [Doctrine ORM Bundle](https://github.com/sonata-project/SonataDoctrineORMAdminBundle) iterates over all the registered entities and if they do not have a  'audit = false' in the admin definition, add it to the registered audit entities. This can be turned of by:
```yaml
# app/config/config.yml
sonata_doctrine_orm_admin:
    audit:
        force: false
```

While this might be an error on behalf of the Doctrine ORM bundle, I think that it'd be wise to filter the locations anyhow. A fix has been proposed in #285.
